ui/app/controllers/vault/cluster/auth.js

```diff
@@ -7,11 +7,32 @@ export default Controller.extend({
   vaultController: controller('vault'),
   clusterController: controller('vault.cluster'),
   namespaceService: service('namespace'),
+  configService: service('config'),
   namespaceQueryParam: alias('clusterController.namespaceQueryParam'),
   queryParams: [{ authMethod: 'with' }],
   wrappedToken: alias('vaultController.wrappedToken'),
   authMethod: '',
   redirectTo: alias('vaultController.redirectTo'),
+  managedNamespaceRoot: alias('configService.managedNamespaceRoot'),
+
+  get managedNamespaceChild() {
+    let fullParam = this.namespaceQueryParam;
+    let split = fullParam.split('/');
+    if (split.length > 1) {
+      split.shift();
+      console.log(split);
+      return `/${split.join('/')}`;
+    }
+    return '';
+  },
+
+  updateManagedNamespace: task(function*(value) {
+    yield timeout(500);
```

Comment:
curious about the need for a timeout here?  maybe add a comment

Reply:
Yes, same reason for the timeout on the manageNamespace fn but I'll copy it over. 

I could also make this DRYer by refactoring the original method but I wanted to not touch as much of the original functioning code as possible. Thoughts?

Reply:
Maybe just add a comment. I agree, that maybe not touch the original function.